### PR TITLE
feat(monitoring): Halyard setup

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: setup/monitoring/stackdriver.md [hunk 1]
```diff
@@ -31,22 +31,59 @@ Stackdriver, please contact us through the Spinnaker Slack channel.
...
+`roles/monitoring.metricWriter` roles enabled. If you don't already have this,
+run the following commands:
 
-If you are on Google Cloud Platform and using the Default Service Account
-credentials then you may need to enable the Monitoring-Write scope.
-This scope is enabled by default, however depending on how you create
-the VMs that you are deploying into, the scope may have been left out.
+```bash
+SERVICE_ACCOUNT_NAME=spinnaker-monitoring-account
+SERVICE_ACCOUNT_DEST=~/.gcp/gce-monitoring-account.json
```
Comment (skim1420) [inline setup/monitoring/stackdriver.md:43]: Does this service account apply only to GCE monitoring? How about Kubernetes or other providers?

Comment (lwander) [reply setup/monitoring/stackdriver.md:43]: That's right, splitting these out by function seems right for these docs - we can always write a codelab with a "super" service account that has all needed roles

File: setup/monitoring/stackdriver.md [hunk 2]
```diff
@@ -31,22 +31,59 @@ Stackdriver, please contact us through the Spinnaker Slack channel.
...
+    --iam-account $SA_EMAIL
+```
+
+Now you have a service account with the correct roles sitting in
+`$SERVICE_ACCOUNT_DEST`.
+
+Finally, run the following hal commands to configure Stackdriver:
+
+```bash
+hal config metric-stores stackddriver edit \
```
Comment (skim1420) [inline setup/monitoring/stackdriver.md]: s/stackddriver/stackdriver

File: setup/monitoring/prometheus.md [hunk 3]
```diff
@@ -68,55 +68,33 @@ as you do for Spinnaker's Gate microservice with its Deck Web UI).
...
+   only way to use Prometheus. To configure this given that `<url`> is the 
+   URL to the Prometheus gateway server, run the following command:
+   ```bash
+   hal config metric-stores prometheus edit --push-gateway=<url>
+   ```
+
+These changes will be picked up by your Spinnaker installation next time you
+run `hal deploy apply`.
+
+Proceed to [configure the metric and dashboard
```
Comment (ttomsu) [inline setup/monitoring/prometheus.md]: Not necessary to say "Keep reading the next section"

**Review by ttomsu — APPROVED** [review]
LGTM

**Pull request merged**